src/libultra/io/contchannelreset.c

```diff
@@ -1,3 +1,27 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/contchannelreset/__osContChannelReset.s")
+s32 __osContChannelReset(OSMesgQueue* mq, s32 count) {
```

Comment:
```suggestion
s32 __osContChannelReset(OSMesgQueue* mq, size_t count) {
```
seems like this is a size of memory to blank out

Reply:
Turns out this also has to be `s32` from its use elsewhere. It's also actually the `channel`, rather than just a bare memory number, so `s32` is a better type after all: it's all over the voice stuff.